OscLib/IOscCalcSterile.cxx

```diff
@@ -0,0 +1,82 @@
+#include "OscLib/IOscCalcSterile.h"
+
+#include <iostream>
+
+namespace osc
+{
+  //---------------------------------------------------------------------------
+  void IOscCalcSterile::SetDmsq21(const double&)
+  {
+    std::cerr << "Must use SetDm!" << std::endl;
+    assert(false);
+  }
+
+  //---------------------------------------------------------------------------
+  void IOscCalcSterile::SetDmsq32(const double&)
+  {
+    std::cerr << "Must use SetDm!" << std::endl;
+    assert(false);
+  }
+
+  //---------------------------------------------------------------------------
+  void IOscCalcSterile::SetTh12(const double&)
+  {
+    std::cerr << "Must use SetAngle!" << std::endl;
+    assert(false);
+  }
+
+  //---------------------------------------------------------------------------
+  void IOscCalcSterile::SetTh13(const double&)
+  {
+    std::cerr << "Must use SetAngle!" << std::endl;
+    assert(false);
+  }
+
+  //---------------------------------------------------------------------------
+  void IOscCalcSterile::SetTh23(const double&)
+  {
+    std::cerr << "Must use SetAngle!" << std::endl;
+    assert(false);
+  }
+
+  //---------------------------------------------------------------------------
+  void IOscCalcSterile::SetdCP(const double&)
+  {
+    std::cerr << "Must use SetDelta!" << std::endl;
+    assert(false);
+  }
+
+  //---------------------------------------------------------------------------
+  double OscCalcSterileTrivial::P(int, int, double)
+  {
+    return 1;
+  }
+
+  //---------------------------------------------------------------------------
+  IOscCalcAdjustable* OscCalcSterileTrivial::Copy() const
+  {
+    return new OscCalcSterileTrivial();
+  }
+
+  //---------------------------------------------------------------------------
+  const IOscCalcSterile* DowncastToSterile(const IOscCalc* calc)
+  {
+    const IOscCalcSterile* calc_sterile
+            = dynamic_cast<const IOscCalcSterile*>(calc);
+    if(calc_sterile) return calc_sterile;
+    std::cout << "Input calculator was not of type IOscCalcSterile." << std::endl;
```

Comment:
Can we add a `bool quiet` parameter (default false?) in case there's some use case that would want to be consistently doing this, and not wanting to spam the console?

Reply:
Done